ufs.c

```diff
@@ -108,7 +112,14 @@ static int parse_args(int argc, char **argv, command_function *func,
 		goto out;
 	}
 
-	rc = init_options(argc, argv, options);
+	if (options->config_type_inx == RPMB_TYPE)
```

Comment:
why breaking from the option convention?
what may cause such parameter confusion that we cannot handle it here?